app/src/main/java/com/kahuis/repairtool/presentation/login/LoginScreen.kt

```diff
@@ -63,43 +61,34 @@ fun LoginView() {
         }
 
         //Get username AND password from loginPackage
-        val uName = singleLineInputLabel("Gebruikersnaam")
+        uName = singleLineInputLabel("Gebruikersnaam")
         Spacer(modifier = Modifier.padding(8.dp))
         val pWord = singelLinePasswordLabel("Wachtwoord")
 
         Spacer(modifier = Modifier.padding(top = 30.dp))
 
         //Login button AND forgot password button
+        /*
+        Op dit moment moet de navigatie en code die hier voor gebruikt wordt nog gemaakt worden.
```

Comment:
It's a common practice to write comments in English, so everyone can read this. Something to think about 👍